api/config/v1alpha1/groupversion_info.go

```diff
@@ -1,7 +1,7 @@
 // Package v1alpha1 contains API Schema definitions for the config v1alpha1 API group.
 //
-//+kubebuilder:object:generate=true
-//+groupName=config.gateway.envoyproxy.io
+// +kubebuilder:object:generate=true
```

Comment:
You must be running go 1.19 since gofmt now adds a space to doc comments. Since the project still uses 1.18, this change should be reverted.